src/base/dbcsr_base_uses.f90

```diff
@@ -49,6 +49,15 @@
 #define TO_VERSION3(MAJOR, MINOR, UPDATE) (TO_VERSION2(MAJOR, MINOR) + (UPDATE))
 #define TO_VERSION TO_VERSION2
 
+! Rely on incorrect expansion inside of quotes to avoid issue of handling hash-character.
+!&<
+#if defined(__GNUC__)
```

Comment:
no sure I understand this part...
GNU preprocessor is different? This code has been used since a while now, is it something with the new GNU versions?

Reply:
wow, just found this https://gcc.gnu.org/legacy-ml/fortran/2009-06/msg00131.html...

Reply:
Well, it quietly failed. You would not note if the applications looks up the wrong environment variable. Also, we do not test so many non-GNU Fortran compilers...